src/wayland/shm/pool.rs

```diff
@@ -17,8 +17,10 @@ use tracing::{debug, instrument, trace};
 
 thread_local!(static SIGBUS_GUARD: Cell<(*const MemMap, bool)> = Cell::new((ptr::null_mut(), false)));
```

Comment:
This file seems to be done, so mark it with the unsafe_op_in_unsafe_fn attribute.

Reply:
Hm... I though that I should fix them step by step until all points have been fixed [here](https://github.com/Smithay/smithay/issues/946), so I'll add `unsafe_op_in_unsafe_fn` to `lib.rs`.

Reply:
Hm... I though that I should fix them step by step until all points have been fixed https://github.com/Smithay/smithay/issues/946, so I'll add unsafe_op_in_unsafe_fn to lib.rs.

Reply:
Not to the lib.rs but this file (`src/wayland/shm/pool.rs`)

Reply:
ok